src/aap_eda/core/management/commands/gather_analytics.py

```diff
@@ -71,8 +71,8 @@ def handle(self, *args, **options):
         opt_since = options.get("since")
         opt_until = options.get("until")
 
-        if not flag_enabled("EDA_ANALYTICS"):
-            self.logger.error("EDA_ANALYTICS is disabled.")
+        if not flag_enabled("FEATURE_EDA_ANALYTICS_ENABLED"):
+            self.logger.error("FEATURE_EDA_ANALYTICS_ENABLED is disabled.")
```

Comment:
Should this say `FEATURE_EDA_ANALYTICS_ENABLED is false`? Having enabled in the name and using enable/disable as a verb seems a bit weird.

Reply:
We can if this makes more sense to folks, In my opinion though `FEATURE_EDA_ANALYTICS_ENABLED is false` seems stranger to me though

Reply:
I would agree with @Dostonbek1 Features are disabled or enabled, flags are True or False. 

IHMO, the message looks better as `Feature eda analytics is disabled.` or `FEATURE_EDA_ANALYTICS_ENABLED is set to False.`

Reply:
Sounds good, I will change it to `FEATURE_EDA_ANALYTICS_ENABLED is set to False.`